utils/utils.go

```diff
@@ -78,6 +82,31 @@ func ToStringList(configured []interface{}) []string {
 	return vs
 }
 
+func CalculateTimeoutOrDefault(ctx context.Context, options *PollingOptions, fallback time.Duration) (*time.Duration, error) {
+	tflog.Debug(ctx, fmt.Sprintf("About to calculate polling timeout using the desired timeout (%+v) and the given fallback timeout (%+v)", options, fallback))
+	var timeout time.Duration
+	if options != nil && !options.PollingTimeout.IsNull() {
+		intPT := int(options.PollingTimeout.ValueInt64())
+		t2, err := time.ParseDuration(strconv.Itoa(intPT) + "m")
```

Comment:
This might work too and more straightforward:
`t2 := time.Duration(options.PollingTimeout.ValueInt64()) * time.Minute`